code/modules/robotics/bot/secbot.dm

```diff
@@ -274,7 +388,7 @@ Report Arrests: <A href='?src=\ref[src];operation=report'>[report_arrests ? "On"
 		if (src.emagged < emag_stages)
 			if (emagged)
```

Comment:
Adjust shot_number on the projectile here, (leaving aside the gameplay concerns of that existing)